guides/dust-helpers.md

```diff
@@ -226,3 +226,17 @@ function yell(chunk, context, bodies, params) {
 }
 dust.helpers.yell = yell;
 ```
+
+## Returning a Value (Dust 2.7.1)
+
+Helpers can return a value directly. When a helper returns a value, by default the value will be HTML-sanitized. You can pass filters to a helper by adding a `filters` param.
+
+    {@myHelper value=name filters="|s|uc" /}
+
+If you need more control over the default behavior (for example if you want to render a body or not HTML-sanitize by default), your helper needs to return the Chunk it's passed instead.
```

Comment:
Can do, thanks.

On Thu, Apr 30, 2015, 8:49 AM Steven <notifications@github.com> wrote:

> In guides/dust-helpers.md
> <https://github.com/linkedin/dustjs/pull/651#discussion_r29441780>:
>
> > @@ -226,3 +226,17 @@ function yell(chunk, context, bodies, params) {
> >  }
> >  dust.helpers.yell = yell;
> >  ```
> > +
> > +## Returning a Value (Dust 2.7.1)
> > +
> > +Helpers can return a value directly. When a helper returns a value, by default the value will be HTML-sanitized. You can pass filters to a helper by adding a `filters` param.
> > +
> > +    {@myHelper value=name filters="|s|uc" /}
> > +
> > +If you need more control over the default behavior (for example if you want to render a body or not HTML-sanitize by default), your helper needs to return the Chunk it's passed instead.
>
> The last part of this sentence is a bit awkward. Maybe it could be
> something like "your helper needs to return the Chunk that was passed to
> it."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linkedin/dustjs/pull/651/files#r29441780>.
>